extensions/wikia/AdEngine/js/lookup/rubiconFastlane.js

```diff
@@ -97,20 +97,17 @@ define('ext.wikia.adEngine.lookup.rubiconFastlane', [
 		});
 	}
 
-	function setTargeting(slotName, targeting, rubiconSlot, provider) {
-		var s1 = context.targeting.wikiIsTop1000 ? adLogicZoneParams.getName() : 'not a top1k wiki';
-		if (targeting) {
-			Object.keys(targeting).forEach(function (key) {
-				rubiconSlot.setFPI(key, targeting[key]);
+	function setTargeting(slotName, slotTargeting, rubiconSlot, skin) {
+		var targeting = rubiconTargeting.getTargeting(slotName, skin, 'fastlane');
```

Comment:
it's possible that I made mistake in reading but `skin` here is `gpt` or `mobile` not `oasis` or `mercury`. `Skin` argument comes from this method -> https://github.com/Wikia/app/pull/11206/files#diff-83eb1fa9c5afed5b4b7d3f13ec99171aR113. Inside this method you set the `provider` variable which then is passed to this method as `skin`